engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
@sanjaypujare
- The CUSTOM_SSL_KEYSTORE_CONFIG setting is only used during application start, so it is not required for the Apex client itself. 
- I don't see why setting/property is preferable over an attribute. Attributes support per application/operator/port specification, while settings do not.
- Why is it OK to blindly overwrite STRAM_HTTP_CUSTOM_CONFIG?